backend/src/routes/ai.py

```diff
@@ -0,0 +1,142 @@
+from routes import app
+from flask import request
+import random
+import pandas as pd #load CSV files
+import numpy as np 
+import seaborn as sns
+import matplotlib.pyplot as plt #plots a graph for each feature
+from sklearn import ensemble
+from sklearn.ensemble import RandomForestRegressor
+from sklearn.svm import SVR
+from sklearn.preprocessing import StandardScaler, LabelEncoder
+from sklearn.metrics import mean_squared_error
+from sklearn.model_selection import GridSearchCV
+import pickle
+
+with open('data\\ai_model.pkl', 'rb') as f:
+    columns = pickle.load(f)
+    model_rf = pickle.load(f)
+
+@app.route('/')
+def home():
+    return '''<a href="/get-house-info?address=615 Reid St, Fredericton, NB, CA">Get House Info</a><br><br>
+    <a href="/get-test-url">Get Test URL</a>  -  <a href="/get-ai-args-test">Get Test AI Args</a><br><br>
+    <a href="/get-ai-url">Get AI Test URL</a>  -  <a href="/get-ai-args">Get AI Args</a>'''
+
+@app.route('/get-house-info')
+def get_house_info():
+    if not 'address' in request.args:
+        return "No address provided, use the format /get-house-info?address='address'"
```

Comment:
I'm not positive but is this not redundant? Idk exaclty how python works but I feel like attempting to .get address in args would return None.

Reply:
im not sure what you mean. That line is there if the address parameter isnt in the request the `'address'` at the end of `address='address'` is the address you want the info for. i didnt want to use `<address>` just incase the HTML did something fucky